### ISSUE TITLE
client connection to com08

### ISSUE THREAD

**Issue by Athanaseus** [ratt-ru/radiopadre-client#23]
I run the following:
```
(radiopadre-client3) athanaseus@athanaseus-E5550:~$ run-radiopadre aramaila@<ip>:/scratch1/aramaila/busyweek/javasan/minimalConfig-docker/output/ --docker
radiopadre.client [0.01s]: Welcome to the radiopadre client!
radiopadre.client [0.01s]:   command line specifies BACKEND = docker
radiopadre.client: Opening ssh connection to aramaila@<ip>. You may be prompted for your password.

radiopadre.client: Using remote client script within /home/aramaila/.radiopadre/venv
radiopadre.client: running radiopadre client on aramaila@<ip>
radiopadre.client: aramaila@<ip>: radiopadre.client [0.02s]: Welcome to the radiopadre client!
radiopadre.client: aramaila@<ip>: radiopadre.client: Using /usr/bin/docker for container mode
radiopadre.client: aramaila@<ip>: radiopadre.client: Starting new session in container radiopadre-aramaila-0f6079a952864ced9ecbc81b8e5b725b
radiopadre.client: aramaila@<ip>: radiopadre.client:     container radiopadre-aramaila-427c8c3411f149e3b92bec29401ea746 is no longer running, clearing up session dir
radiopadre.client: aramaila@<ip>: radiopadre.client:   Selected ports: 1024:1025:1026:1039:1040:13001:13002:13003:13004:13005
radiopadre.client: aramaila@<ip>: radiopadre.client:   Session ID/notebook token is '1cecfa7a6d0b4d54a480debbed926af5'
radiopadre.client: aramaila@<ip>: radiopadre.client:   Container name: radiopadre-aramaila-0f6079a952864ced9ecbc81b8e5b725b
radiopadre.client: aramaila@<ip>: radiopadre.client:   Using radiopadre docker image osmirnov/radiopadre:1.0pre10
radiopadre.client: aramaila@<ip>: radiopadre.client:   Available notebooks: mypipelinerun-project-logs.ipynb mypipelinerun-std-progress-report.ipynb mypipelinerun-detailed-final-report.ipynb
radiopadre.client: aramaila@<ip>: radiopadre.client:   No notebooks matching --auto-load radiopadre-auto.ipynb
radiopadre.client: aramaila@<ip>: radiopadre.client: Browse to URL: http://localhost:13001/?token=1cecfa7a6d0b4d54a480debbed926af5
radiopadre.client: aramaila@<ip>: radiopadre.client: Browse to URL: http://localhost:13004/?socketUrl=ws://localhost:13005 (CARTA file browser)
radiopadre.client: aramaila@<ip>: radiopadre.client: Container name: radiopadre-aramaila-0f6079a952864ced9ecbc81b8e5b725b
radiopadre.client: aramaila@<ip>: radiopadre.client: Running /usr/bin/docker run --rm --name radiopadre-aramaila-0f6079a952864ced9ecbc81b8e5b725b -w /scratch1/aramaila/busyweek/javasan/minimalConfig-docker/output --user 1801:1801 -e USER=aramaila -e HOME=/home/aramaila -e RADIOPADRE_CONTAINER_NAME=radiopadre-aramaila-0f6079a952864ced9ecbc81b8e5b725b -e RADIOPADRE_SESSION_ID=1cecfa7a6d0b4d54a480debbed926af5 -d -p 1024:11001/tcp -p 1025:11002/tcp -p 1026:11003/tcp -p 1039:11004/tcp -p 1040:11005/tcp -v /scratch1/aramaila/busyweek/javasan/minimalConfig-docker/output:/scratch1/aramaila/busyweek/javasan/minimalConfig-docker/output -v /home/aramaila:/home/aramaila -v /home/aramaila/.radiopadre/.docker-local:/home/aramaila/.local -v /home/aramaila/.radiopadre/sessions/radiopadre-aramaila-0f6079a952864ced9ecbc81b8e5b725b:/scratch1/aramaila/busyweek/javasan/minimalConfig-docker/output/.radiopadre-session -v /home/aramaila/.radiopadre/sessions/radiopadre-aramaila-0f6079a952864ced9ecbc81b8e5b725b:/home/aramaila/.radiopadre/scratch1/aramaila/busyweek/javasan/minimalConfig-docker/output/.radiopadre-session -v /home/aramaila/.radiopadre/.js9-tmp:/.radiopadre/venv/js9-www/tmp --label radiopadre.user=aramaila --label radiopadre.dir=/scratch1/aramaila/busyweek/javasan/minimalConfig-docker/output osmirnov/radiopadre:1.0pre10 run-radiopadre --auto-load radiopadre-auto.ipynb --backend venv --browser None --carta-browser --default-notebook radiopadre-default.ipynb --docker-image osmirnov/radiopadre:1.0pre10 --grim-reaper --client-install-repo  --client-install-branch b1.0-pre10 --client-install-pip radiopadre-client --server-install-repo  --server-install-branch b1.0-pre10 --server-install-pip radiopadre --verbose 0 --radiopadre-venv /.radiopadre/venv --inside-container 11001:11002:11003:11004:11005:13001:13002:13003:13004:13005 --workdir /home/aramaila/.radiopadre .
radiopadre.client: aramaila@<ip>: radiopadre.client: Waiting for up to 60 secs for the notebook container to come up
radiopadre.client: aramaila@<ip>: radiopadre.client: Container started. The jupyter notebook server is running on port 1024 (after 2.41 secs)
radiopadre.client: Running xdg-open http://localhost:13001/?token=1cecfa7a6d0b4d54a480debbed926af5 http://localhost:13004/?socketUrl=ws://localhost:13005
radiopadre.client:   if this fails, specify a correct browser invocation command with --browser and rerun,
radiopadre.client:   or else browse to the URL given above ("Browse to URL:") yourself.
radiopadre.client: The remote radiopadre session is now fully up
radiopadre.client: Press Ctrl+C to kill the remote session
```

And then the popping browser says, site can’t be reached:
```
This site can’t be reached
The connection was reset.
Try:

Checking the connection
Checking the proxy and the firewall
ERR_CONNECTION_RESET
```
NB: <ip> address is censored.
PS: The connection is successfully established for stills which uses singularity

**Comment by o-smirnov** [comment]
Ok, first re-run with ``-u``, just to make sure it's not an old install on com08 screwing with you.

If it still fails, could you please run with ``--log -v 2 --container-debug``? That should be more informative.

**Comment by Athanaseus** [comment]
```
(radiopadre-client3) athanaseus@athanaseus-E5550:~$ run-radiopadre aramaila@<ip>:/scratch1/aramaila/busyweek/javasan/minimalConfig-docker/output/ --docker -u
radiopadre.client [0.01s]: Welcome to the radiopadre client!
radiopadre.client [0.01s]:   command line specifies BACKEND = docker
radiopadre.client [0.01s]:   command line specifies UPDATE = True
radiopadre.client: Opening ssh connection to aramaila@<ip>. You may be prompted for your password.
radiopadre.client: Using remote client script within /home/aramaila/.radiopadre/venv
radiopadre.client: --client-install-pip radiopadre-client is configured.
radiopadre.client: Will attempt to pip install -U radiopadre-client in aramaila@<ip>:~/.radiopadre/venv
WARNING: You are using pip version 20.0.2; however, version 20.1 is available.
You should consider upgrading via the '/home/aramaila/.radiopadre/venv/bin/python3 -m pip install --upgrade pip' command.
radiopadre.client: Success!
radiopadre.client: running radiopadre client on aramaila@<ip>
radiopadre.client: aramaila@<ip>: radiopadre.client [0.01s]: Welcome to the radiopadre client!
radiopadre.client: aramaila@<ip>: radiopadre.client: Using /usr/bin/docker for container mode
radiopadre.client: aramaila@<ip>: radiopadre.client: Starting new session in container radiopadre-aramaila-2bfe6163050b4201a3ece24e1d78a92e
radiopadre.client: aramaila@<ip>: radiopadre.client:     container radiopadre-aramaila-03d45d31ea2548e9b694dc1e542ee9fc is no longer running, clearing up session dir
radiopadre.client: aramaila@<ip>: radiopadre.client:   Selected ports: 1024:1025:1026:1039:1040:13006:13007:13008:13009:13010
radiopadre.client: aramaila@<ip>: radiopadre.client:   Session ID/notebook token is '835ceb988a7c44febba6d16e7438379e'
radiopadre.client: aramaila@<ip>: radiopadre.client:   Container name: radiopadre-aramaila-2bfe6163050b4201a3ece24e1d78a92e
radiopadre.client: aramaila@<ip>: radiopadre.client: True None .
radiopadre.client: aramaila@<ip>: radiopadre.client:   Using radiopadre docker image osmirnov/radiopadre:1.0pre10
radiopadre.client: WARNING: aramaila@<ip>: radiopadre.client: WARNING: --update or --auto-init specified: calling docker pull to make sure the image is up-to-date.
radiopadre.client: WARNING: aramaila@<ip>: radiopadre.client: WARNING:   (This may take a few minutes if it isn't....)
radiopadre.client: aramaila@<ip>: 1.0pre10: Pulling from osmirnov/radiopadre
radiopadre.client: aramaila@<ip>: Digest: sha256:1f7ba20b67adacb8fb5e59c01fd1142f0093a5c6eadb8ef719f3a928daf01d9e
radiopadre.client: aramaila@<ip>: Status: Image is up to date for osmirnov/radiopadre:1.0pre10
radiopadre.client: aramaila@<ip>: docker.io/osmirnov/radiopadre:1.0pre10
radiopadre.client: aramaila@<ip>: radiopadre.client:   Available notebooks: mypipelinerun-project-logs.ipynb mypipelinerun-std-progress-report.ipynb mypipelinerun-detailed-final-report.ipynb
radiopadre.client: aramaila@<ip>: radiopadre.client:   No notebooks matching --auto-load radiopadre-auto.ipynb
radiopadre.client: aramaila@<ip>: radiopadre.client: Browse to URL: http://localhost:13006/?token=835ceb988a7c44febba6d16e7438379e
radiopadre.client: aramaila@<ip>: radiopadre.client: Browse to URL: http://localhost:13009/?socketUrl=ws://localhost:13010 (CARTA file browser)
radiopadre.client: aramaila@<ip>: radiopadre.client: Container name: radiopadre-aramaila-2bfe6163050b4201a3ece24e1d78a92e
radiopadre.client: aramaila@<ip>: radiopadre.client: Running /usr/bin/docker run --rm --name radiopadre-aramaila-2bfe6163050b4201a3ece24e1d78a92e -w /scratch1/aramaila/busyweek/javasan/minimalConfig-docker/output --user 1801:1801 -e USER=aramaila -e HOME=/home/aramaila -e RADIOPADRE_CONTAINER_NAME=radiopadre-aramaila-2bfe6163050b4201a3ece24e1d78a92e -e RADIOPADRE_SESSION_ID=835ceb988a7c44febba6d16e7438379e -d -p 1024:11001/tcp -p 1025:11002/tcp -p 1026:11003/tcp -p 1039:11004/tcp -p 1040:11005/tcp -v /scratch1/aramaila/busyweek/javasan/minimalConfig-docker/output:/scratch1/aramaila/busyweek/javasan/minimalConfig-docker/output -v /home/aramaila:/home/aramaila -v /home/aramaila/.radiopadre/.docker-local:/home/aramaila/.local -v /home/aramaila/.radiopadre/sessions/radiopadre-aramaila-2bfe6163050b4201a3ece24e1d78a92e:/scratch1/aramaila/busyweek/javasan/minimalConfig-docker/output/.radiopadre-session -v /home/aramaila/.radiopadre/sessions/radiopadre-aramaila-2bfe6163050b4201a3ece24e1d78a92e:/home/aramaila/.radiopadre/scratch1/aramaila/busyweek/javasan/minimalConfig-docker/output/.radiopadre-session -v /home/aramaila/.radiopadre/.js9-tmp:/.radiopadre/venv/js9-www/tmp --label radiopadre.user=aramaila --label radiopadre.dir=/scratch1/aramaila/busyweek/javasan/minimalConfig-docker/output osmirnov/radiopadre:1.0pre10 run-radiopadre --auto-load radiopadre-auto.ipynb --backend venv --browser None --carta-browser --default-notebook radiopadre-default.ipynb --docker-image osmirnov/radiopadre:1.0pre10 --grim-reaper --client-install-repo  --client-install-branch b1.0-pre10 --client-install-pip radiopadre-client --server-install-repo  --server-install-branch b1.0-pre10 --server-install-pip radiopadre --verbose 0 --radiopadre-venv /.radiopadre/venv --inside-container 11001:11002:11003:11004:11005:13006:13007:13008:13009:13010 --workdir /home/aramaila/.radiopadre .
radiopadre.client: aramaila@<ip>: radiopadre.client: Container started. The jupyter notebook server is running on port 1024 (after 1.00 secs)
radiopadre.client: Running xdg-open http://localhost:13006/?token=835ceb988a7c44febba6d16e7438379e http://localhost:13009/?socketUrl=ws://localhost:13010
radiopadre.client:   if this fails, specify a correct browser invocation command with --browser and rerun,
radiopadre.client:   or else browse to the URL given above ("Browse to URL:") yourself.
radiopadre.client: The remote radiopadre session is now fully up
radiopadre.client: Press Ctrl+C to kill the remote session
```

**Comment by Athanaseus** [comment]
```
(radiopadre-client3) athanaseus@athanaseus-E5550:~$ run-radiopadre aramaila@<ip>:/scratch1/aramaila/busyweek/javasan/minimalConfig-docker/output/ --docker --log -v 2 --container-debug
radiopadre.client [0.01s]: Welcome to the radiopadre client!
radiopadre.client [0.01s]:   command line specifies CONTAINER_DEBUG = True
radiopadre.client [0.01s]:   command line specifies BACKEND = docker
radiopadre.client [0.01s]:   command line specifies LOG = True
radiopadre.client [0.01s]:   command line specifies VERBOSE = 2
radiopadre.client: writing session log to /home/athanaseus/.radiopadre/logs/log-local-20200504155519.txt
radiopadre.client: DEBUG: PID is 24785
radiopadre.client: Opening ssh connection to aramaila@<ip>. You may be prompted for your password.
radiopadre.client: DEBUG:   ssh -o ControlPath=/tmp/ssh_mux_radiopadre_%C -o ControlMaster=auto -o ControlPersist=1h aramaila@<ip>
radiopadre.client: Checking installation on aramaila@<ip>.
radiopadre.client: Using remote client script within /home/aramaila/.radiopadre/venv
radiopadre.client: running ssh -o ControlPath=/tmp/ssh_mux_radiopadre_%C -o ControlMaster=auto -o ControlPersist=1h aramaila@<ip> shopt -s huponexit && source /home/aramaila/.radiopadre/venv/bin/activate && run-radiopadre --auto-load 'radiopadre-auto.ipynb' --backend 'docker' --browser 'None' --carta-browser --default-notebook 'radiopadre-default.ipynb' --docker-image 'osmirnov/radiopadre:1.0pre10' --container-debug --grim-reaper --client-install-path '~/radiopadre-client' --client-install-repo '' --client-install-branch 'b1.0-pre10' --client-install-pip 'radiopadre-client' --server-install-path '~/radiopadre' --server-install-repo '' --server-install-branch 'b1.0-pre10' --server-install-pip 'radiopadre' --verbose '2' --log --radiopadre-venv '~/.radiopadre/venv' --remote '13011:13012:13013:13014:13015' '/scratch1/aramaila/busyweek/javasan/minimalConfig-docker/output/' 
radiopadre.client: aramaila@<ip>: radiopadre.client [0.01s]: Welcome to the radiopadre client!
radiopadre.client: aramaila@<ip>: radiopadre.client: writing session log to /home/aramaila/.radiopadre/logs/log-remote-20200504135520.txt
radiopadre.client: DEBUG: aramaila@<ip>: radiopadre.client: DEBUG: PID is 36882
radiopadre.client: aramaila@<ip>: radiopadre.client: Using /usr/bin/docker for container mode
radiopadre.client: aramaila@<ip>: radiopadre.client: Starting new session in container radiopadre-aramaila-975e577dd64a41659ec876c927adabe5
radiopadre.client: aramaila@<ip>: radiopadre.client:     container radiopadre-aramaila-2bfe6163050b4201a3ece24e1d78a92e is no longer running, clearing up session dir
radiopadre.client: aramaila@<ip>: radiopadre.client:   Selected ports: 1024:1025:1026:1039:1040:13011:13012:13013:13014:13015
radiopadre.client: Detected ports 1024:1025:1026:1039:1040 -> 13011:13012:13013:13014:13015
radiopadre.client: sending forward request to ssh mux process
radiopadre.client: aramaila@<ip>: radiopadre.client:   Session ID/notebook token is '44a08daa9fe14f3abc6317907b1f0e2e'
radiopadre.client: aramaila@<ip>: radiopadre.client:   Container name: radiopadre-aramaila-975e577dd64a41659ec876c927adabe5
radiopadre.client: aramaila@<ip>: radiopadre.client:   Remote mode -- not opening a browser locally.
radiopadre.client: aramaila@<ip>: radiopadre.client: True None .
radiopadre.client: DEBUG: aramaila@<ip>: radiopadre.client: DEBUG: Setting up radiopadre environment, cwd is /scratch1/aramaila/busyweek/javasan/minimalConfig-docker/output
radiopadre.client: DEBUG: aramaila@<ip>: radiopadre.client: DEBUG:   setting SERVER_BASEDIR to /scratch1/aramaila/busyweek/javasan/minimalConfig-docker/output
radiopadre.client: aramaila@<ip>: radiopadre.client:   Using radiopadre docker image osmirnov/radiopadre:1.0pre10
radiopadre.client: aramaila@<ip>: radiopadre.client:   Available notebooks: mypipelinerun-project-logs.ipynb mypipelinerun-std-progress-report.ipynb mypipelinerun-detailed-final-report.ipynb
radiopadre.client: aramaila@<ip>: radiopadre.client:   No notebooks matching --auto-load radiopadre-auto.ipynb
radiopadre.client: aramaila@<ip>: radiopadre.client: Browse to URL: http://localhost:13011/?token=44a08daa9fe14f3abc6317907b1f0e2e
radiopadre.client: aramaila@<ip>: radiopadre.client: Browse to URL: http://localhost:13014/?socketUrl=ws://localhost:13015 (CARTA file browser)
radiopadre.client: aramaila@<ip>: radiopadre.client: Container name: radiopadre-aramaila-975e577dd64a41659ec876c927adabe5
radiopadre.client: aramaila@<ip>: radiopadre.client: Running /usr/bin/docker run --rm --name radiopadre-aramaila-975e577dd64a41659ec876c927adabe5 -w /scratch1/aramaila/busyweek/javasan/minimalConfig-docker/output --user 1801:1801 -e USER=aramaila -e HOME=/home/aramaila -e RADIOPADRE_CONTAINER_NAME=radiopadre-aramaila-975e577dd64a41659ec876c927adabe5 -e RADIOPADRE_SESSION_ID=44a08daa9fe14f3abc6317907b1f0e2e -p 1024:11001/tcp -p 1025:11002/tcp -p 1026:11003/tcp -p 1039:11004/tcp -p 1040:11005/tcp -v /scratch1/aramaila/busyweek/javasan/minimalConfig-docker/output:/scratch1/aramaila/busyweek/javasan/minimalConfig-docker/output -v /home/aramaila:/home/aramaila -v /home/aramaila/.radiopadre/.docker-local:/home/aramaila/.local -v /home/aramaila/.radiopadre/sessions/radiopadre-aramaila-975e577dd64a41659ec876c927adabe5:/scratch1/aramaila/busyweek/javasan/minimalConfig-docker/output/.radiopadre-session -v /home/aramaila/.radiopadre/sessions/radiopadre-aramaila-975e577dd64a41659ec876c927adabe5:/home/aramaila/.radiopadre/scratch1/aramaila/busyweek/javasan/minimalConfig-docker/output/.radiopadre-session -v /home/aramaila/.radiopadre/.js9-tmp:/.radiopadre/venv/js9-www/tmp --label radiopadre.user=aramaila --label radiopadre.dir=/scratch1/aramaila/busyweek/javasan/minimalConfig-docker/output osmirnov/radiopadre:1.0pre10 run-radiopadre --auto-load radiopadre-auto.ipynb --backend venv --browser None --carta-browser --default-notebook radiopadre-default.ipynb --docker-image osmirnov/radiopadre:1.0pre10 --container-debug --grim-reaper --client-install-repo  --client-install-branch b1.0-pre10 --client-install-pip radiopadre-client --server-install-repo  --server-install-branch b1.0-pre10 --server-install-pip radiopadre --verbose 2 --log --radiopadre-venv /.radiopadre/venv --inside-container 11001:11002:11003:11004:11005:13011:13012:13013:13014:13015 --workdir /home/aramaila/.radiopadre .
radiopadre.client: aramaila@<ip>: radiopadre.client: Container started. The jupyter notebook server is running on port 1024 (after 1.00 secs)
radiopadre.client: Running xdg-open http://localhost:13011/?token=44a08daa9fe14f3abc6317907b1f0e2e http://localhost:13014/?socketUrl=ws://localhost:13015
radiopadre.client:   if this fails, specify a correct browser invocation command with --browser and rerun,
radiopadre.client:   or else browse to the URL given above ("Browse to URL:") yourself.
radiopadre.client: The remote radiopadre session is now fully up
radiopadre.client: Press Ctrl+C to kill the remote session
radiopadre.client: aramaila@<ip> stderr: usage: run-radiopadre [-h] [-b COMMAND[:OPT[:OPT...]]] [-n]
```

**Comment by Athanaseus** [comment]
No luck. Did i execute the latter correctly?

**Comment by o-smirnov** [comment]
I'm pretty sure you have radiopadre-client checked out on the remote end, and that it's out of date. Have you tried ``-u``?

This is the culprit:

```
radiopadre.client: aramaila@<ip>: radiopadre.client: Running /usr/bin/docker run --rm --name radiopadre-aramaila-975e577dd64a41659ec876c927adabe5 -w /scratch1/aramaila/busyweek/javasan/minimalConfig-docker/output --user 1801:1801 -e USER=aramaila -e HOME=/home/aramaila -e RADIOPADRE_CONTAINER_NAME=radiopadre-aramaila-975e577dd64a41659ec876c927adabe5 -e RADIOPADRE_SESSION_ID=44a08daa9fe14f3abc6317907b1f0e2e -p 1024:11001/tcp -p 1025:11002/tcp -p 1026:11003/tcp -p 1039:11004/tcp -p 1040:11005/tcp -v /scratch1/aramaila/busyweek/javasan/minimalConfig-docker/output:/scratch1/aramaila/busyweek/javasan/minimalConfig-docker/output -v /home/aramaila:/home/aramaila -v /home/aramaila/.radiopadre/.docker-local:/home/aramaila/.local -v /home/aramaila/.radiopadre/sessions/radiopadre-aramaila-975e577dd64a41659ec876c927adabe5:/scratch1/aramaila/busyweek/javasan/minimalConfig-docker/output/.radiopadre-session -v /home/aramaila/.radiopadre/sessions/radiopadre-aramaila-975e577dd64a41659ec876c927adabe5:/home/aramaila/.radiopadre/scratch1/aramaila/busyweek/javasan/minimalConfig-docker/output/.radiopadre-session -v /home/aramaila/.radiopadre/.js9-tmp:/.radiopadre/venv/js9-www/tmp --label radiopadre.user=aramaila --label radiopadre.dir=/scratch1/aramaila/busyweek/javasan/minimalConfig-docker/output osmirnov/radiopadre:1.0pre10 run-radiopadre --auto-load radiopadre-auto.ipynb --backend venv --browser None --carta-browser --default-notebook radiopadre-default.ipynb --docker-image osmirnov/radiopadre:1.0pre10 --container-debug --grim-reaper --client-install-repo  --client-install-branch b1.0-pre10 --client-install-pip radiopadre-client --server-install-repo  --server-install-branch b1.0-pre10 --server-install-pip radiopadre --verbose 2 --log --radiopadre-venv /.radiopadre/venv --inside-container 11001:11002:11003:11004:11005:13011:13012:13013:13014:13015 --workdir /home/aramaila/.radiopadre .
```
...I have run-radiopadre baked in as the container entry point in the latest rc10. So the client shouldn't be adding its own run-radiopadre to the command. Ergo, your client is out of date.

**Comment by Athanaseus** [comment]
Yes, you are right. 

pip list gave me:
```
radiopadre         1.0rc8
radiopadre-client  1.0rc9
```

But even when i used `-u` it didn't help.
I had to remove the `.radiopadre` in its entirety and do `--auto-init`.
This also gave me `radiopadre-client  1.0rc9` still.
I had to activate the venv and just ` pip install git+https://github.com/ratt-ru/radiopadre-client.git@b1.0-pre10`.

Now it works.

PS: my local `radiopadre-client` up-to date.
